### ISSUE TITLE
HTTP error 404

### ISSUE THREAD

**Issue by florentin68** [maniac103/squeezeclient#10]
I wanted to try the app because with Squeezer I don't manage to connect from outside my home network.
But with Squeeze Client I'm unable to connect internaly nor externaly: I allways get a 404 error.

Any idea to help me?

Thank you in advance.

LMS version: 9.0.1 (30 Dec 2024)
Squeeze Client: 1.4 (F-Droid)

**Comment by maniac103** [comment]
Server URL (including port number) is correct? If you enable server side logging (network.http in debug level), do you see the app access attempts?

**Comment by florentin68** [comment]
Ok I had a middleware in my Traefik configuration which blocked the request.
On local network, my server is found with reverse proxy IP not service IP. I have to switch WiFi OFF so I the IP field is not greyed and I can write the right IP.

I enabled authentification in LMS but then Squeeze Client can't get server anymore.
I understand that the app requests HTTP on port 9000, not HTTPS.

My setup (works when auth not enabled):
Traefik with port 9000 redirected to 443 (https).
LMS reachable to https://lms.mydomain.com or http://lms.mydomain.com:9000.
lms.mydomain.com points to LMS docker service.
Ports 9000 / 9090 / 9483 (TCP and UDP) are in host mode on Docker service (IP2), Traefik passes these ports too (IP1) to the service. These ports are open (IPV4 and IPV6) on my router.

**Comment by florentin68** [comment]
> Server URL (including port number) is correct? If you enable server side logging (network.http in debug level), do you see the app access attempts?

Yes, I saw POST requests to /cometd in Traefik access logs on endpoint port 9000.

**Comment by maniac103** [comment]
> On local network, my server is found with reverse proxy IP not service IP. I have to switch WiFi OFF so I the IP field is not greyed and I can write the right IP.

You can always choose 'Enter server details' as server to enter custom data.

> I enabled authentification in LMS but then Squeeze Client can't get server anymore.

What exactly does that mean? A server log would be helpful here.

**Comment by maniac103** [comment]
Closing due to no response. Feel free to reopen with server log if issue persists.
